hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestSparkSqlCoreFlow.scala

```diff
@@ -29,13 +29,12 @@ import org.apache.spark.sql.{Dataset, Row}
 import org.apache.spark.sql.hudi.HoodieSparkSqlTestBase
 import org.junit.jupiter.api.Assertions.{assertEquals, assertTrue}
 import org.apache.hudi.common.testutils.RawTripTestPayload.recordsToStrings
-import org.junit.jupiter.api.Disabled
 import org.scalatest.Inspectors.forAll
 
 import java.io.File
 import scala.collection.JavaConversions._
 
-@Disabled("HUDI-5304 Disabling to avoid CI timing out")
+@SparkSQLCoreFlow
```

Comment:
Can you help me understand what we're aiming for w/ this new annotation?

Reply:
So @Disabled doesn't seem to work. If you look at the logs for the job that times out and scroll to the bottom, you can see that the sql tests are still running. I looked up how to disable scalatests and @Ignore will disable it but you can't even run the tests locally without removing that tag. The way to disable it seemed to be to tag it and then in the pom put that we should ignore the tag

Reply:
Oh, i misread POM config which excludes it. Makes sense.
We disable this tests temporarily until we rebalance the Azure jobs, right?